src/main/java/frc/utils/dashboard/LoggedTableBoolean.java

```diff
@@ -0,0 +1,60 @@
+package frc.utils.dashboard;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+import org.littletonrobotics.junction.networktables.LoggedDashboardInput;
+
+public class LoggedTableBoolean implements LoggedDashboardInput {
```

Comment:
try to make the value appear in a tab in dashboard too, but if its very complicated its not a must